packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -259,6 +259,10 @@ <h2>
                                                 data-ng-required="$ctrl.httpHeader[$index].model.permission"
                                                 data-ng-disabled="$ctrl.httpHeader[$index].added"
                                             >
+                                                <option
+                                                    data-ng-bind="'pci_projects_project_training_jobs_submit_data_permission_select' | translate"
+                                                    default
```

Comment:
**issue:** Attribute `default` not allowed on element `option`.

Reply:
We have to use "selected" instead.
I saw that we have this in others HTML file, plz update them.

thanks.

Reply:
done